docs/directives/tooltip.md

```diff
@@ -0,0 +1,50 @@
+```js static
+import Tooltip from '@nextcloud/vue/dist/Directives/Tooltip.js'
+```
+
+Tooltip directive based on [v-tooltip](https://github.com/Akryum/v-tooltip).
+
+Note: it's preferred to use [native HTML title attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title) instead for accessibility.
```

Comment:
should we deprecate it?

Reply:
I dunno, to be honest ⚆_⚆

In theory - yes, because we migrated everywhere in nextcloud-vue already to `title`. But apps are still using it also because of visual result.

Maybe we should re-check this rule.

Reply:
But we can do it after mergin docs for already created and not yet marked deprecated directives 👀

Reply:
Yes sure :)